logfields/common.go

```diff
@@ -0,0 +1,24 @@
+package logfields
+
+const (
+	FunctionName = "func"
+	TokenID      = "token_id"
```

Comment:
We have different token Ids for different types of NFT's it may be nice to specify in case you are trying to log both.

```suggestion
	VehicleTokenID      = "vehicle_token_id"
	AftermarketTokenID      = "aftermarket_token_id"
```

Reply:
i like